cpp/src/io/parquet/experimental/hybrid_scan_chunking.cu

```diff
@@ -165,6 +164,16 @@ void hybrid_scan_reader_impl::setup_next_pass(std::vector<rmm::device_buffer> co
       decomp_dict_data_size +
       thrust::reduce(rmm::exec_policy(_stream), chunk_iter, chunk_iter + pass.chunks.size());
 
+    // if we are doing subpass reading, generate more accurate num_row estimates for list columns.
```

Comment:
These lines are copy pasted from the `reader_impl_chunking.cu` to generate estimated or actual list column row counts here